### DISCUSSION TITLE
Automation detection based on the OS

### DISCUSSION THREAD

**Discussion by benfrankie** [seleniumbase/SeleniumBase#2912] [Q&A] [answered]
Hi,

I'm facing an issue with my social media automation scripts getting banned quickly. Recently, accounts started getting banned almost instantly on Linux after a major update by Cloudflare. I moved to a Windows server, but the issue persists. However, accounts seem to last longer when running the scripts on a Windows PC in the same country as the proxies.

Is there a known solution or any recommended adjustments for this issue? Does the server's OS or location significantly affect detection rates?

Thanks for your help.

**Comment by mdmintz** [comment, marked as answer]
Unusually high traffic from an IP address will generally cause that IP to be banned. It’s defense against DDoS. Probably Linux gets banned first because servers generally use that. As far as adjusting your scripts, that’s what I know. Adjust your scripts accordingly. You’ll probably find more information online about the specifics.
